sources/api/schnauzer/src/lib.rs

```diff
@@ -131,6 +131,10 @@ pub fn build_template_registry() -> Result<handlebars::Handlebars<'static>> {
         "kube_reserve_memory",
         Box::new(helpers::kube_reserve_memory),
     );
+    template_registry.register_helper(
+        "add_unresolvable_hostname",
```

Comment:
Ugh, we now have several each of kabob case and snake case. Maybe we should add a comment that specifies what the preference is and that we messed it up.